config/preview.json

```diff
@@ -2,5 +2,8 @@
 	"extends": "base",
 	"io_sites": {
 		"max_static_paths": 10
+	},
+	"flags": {
+		"enable_unified_search": true
```

Comment:
Toggling "unified search" on in preview to make review easier. Have also toggled this on in `development.json`.

🚧 Intent is to toggle these both `preview` and `development` to `false` again before merging to `main`. (If we forget, no big stress cause `production` is `false` either way, but don't want other previews to have a half-baked search experience).